### PR TITLE
Contributor docs

### PR DESCRIPTION
**Pull request by lmccart**
Hello, I've done a pass on updating the readme and contributor_docs, here are the main changes:
* Updating language in readme
* Re-organizing and updating contributor_docs/readme for better clarity
* Adding notes about stewards to both files
* Updating out of date links

Please have a look. Because there was some significant rearrangement, it may be a little hard to follow parts of this. The main area to review is the stewards section in both the `readme` and `contributor_docs/readme`. I can also just merge it in and ask for your review at that time if it's easier.

cc @limzykenneth @stalgiag @outofambit 

Thanks!

### CODE REVIEW

**Comment by stalgiag** [comment]
@lmccart this is really great. I think both readmes are much warmer and easier to follow. And the section about becoming/being a steward is really inviting, concise, and informative. 🙌 🙌 🙌

File: contributor_docs/README.md [hunk 1]
```diff
@@ -140,27 +111,85 @@ A complete guide to unit testing is beyond the scope of the p5.js documentation,
...
+
+* Do not compare x == true or x == false. Use (x) or (!x) instead. x == true, is certainly different from if (x)! Compare objects to null, numbers to 0 or strings to "", if there is chance for confusion.
+
+* Comment your code whenever there is ambiguity or complexity in the function you are writing
+
+* See the [Mozilla JS practices](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#JavaScript_practices) as a useful guide for more styling tips
+
+  
+
+## Unit Tests
```
Comment (outofambit) [inline contributor_docs/README.md]: this section is great and I think parts of it could also go in `unit_testing.md`. after this is merged i'll take a look and propose any revisions i think are worthwhile.

File: contributor_docs/README.md [hunk 2]
```diff
@@ -1,107 +1,78 @@
...
-$ grunt
-```
-
-Sometimes it is useful to run the tests in the browser instead of on the command line. To do this, first start the [connect](https://github.com/gruntjs/grunt-contrib-connect) server:
+# Development Process
 
-```
-$ npm run dev
-```
+We node the development process can be a little tricky at first. You're not alone, it's confusing for everyone at the beginning. The steps below walk you through the setup process. If you have questions, you can ask on the [forum](https://discourse.processing.org/c/p5js) or post an [issue](https://github.com/processing/p5.js/issues) that describes the place you are stuck, and we'll do our best to help.
```
Comment (limzykenneth) [inline contributor_docs/README.md]:
```suggestion
We know the development process can be a little tricky at first. You're not alone, it's confusing for everyone at the beginning. The steps below walk you through the setup process. If you have questions, you can ask on the [forum](https://discourse.processing.org/c/p5js) or post an [issue](https://github.com/processing/p5.js/issues) that describes the place you are stuck, and we'll do our best to help.
```

File: contributor_docs/README.md [hunk 3]
```diff
@@ -140,27 +111,85 @@ A complete guide to unit testing is beyond the scope of the p5.js documentation,
...
-- [website](https://github.com/processing/p5.js-website) This repository contains most of the code for the [p5.js website](http://p5js.org), with the exception of the reference manual. It is maintained by [Lauren McCarthy](https://github.com/lmccart).
-- [sound](https://github.com/processing/p5.js-sound) This repository contains the p5.sound.js library. It is maintained by [Jason Sigal](https://github.com/therewasaguy).
-- [web editor](https://github.com/processing/p5.js-web-editor): This repository contains the source code for the [p5.js web editor](https://editor.p5js.org). It is maintained by [Cassie Tarakajian](https://github.com/catarak). Note that the older [p5.js editor](https://github.com/processing/p5.js-editor) is now deprecated.
 
 # Miscellaneous
 
+- There are a other files in the [contributor_docs/](https://github.com/processing/p5.js/tree/master/contributor_docs) folder that you might explore. They pertain to contributing to specific areas of this project, both technical and non-technical.
```
Comment (limzykenneth) [inline contributor_docs/README.md]:
```suggestion
- There are other files in the [contributor_docs/](https://github.com/processing/p5.js/tree/master/contributor_docs) folder that you might explore. They pertain to contributing to specific areas of this project, both technical and non-technical.
```

**Comment by limzykenneth** [comment]
Just a couple typos but looks good to me overall!

**Comment by limzykenneth** [comment]
Might be for another PR. 

There's an influx of new interest in contributing recently which brings up a couple issues that I think is worth documenting somewhere, to say that

1. How the process of claiming an issue should work (an issue once assigned, if the assignee has not responded for 30 days then the issue can be requested to be reassigned to someone else)
2. To value opinions more than code contributions. I see a lot of interest in just diving into an issue and jump straight to submitting a PR which may not achieve what we need to solve the issue, while it can often be fully solved if it has been worked out in the issue beforehand. And also sometimes the issue is more about asking for user opinion and experience, because we don't have all the answers.

**Comment by outofambit** [comment]
> To value opinions more than code contributions

@limzykenneth big +1 to that

**Comment by stalgiag** [comment]
Yes! Thank you for thinking to add that to this conversation @limzykenneth
